WebHostLib/tracker.py

```diff
@@ -900,6 +900,42 @@ def __renderSuperMetroidTracker(multisave: Dict[str, Any], room: Room, locations
                             checks_done=checks_done, checks_in_area=checks_in_area, location_info=location_info,
                             **display_data)
 
+
+def __renderBingoTracker(multisave: Dict[str, Any], room: Room, locations: Dict[int, Dict[int, Tuple[int, int, int]]],
+                                inventory: Counter, team: int, player: int, playerName: str,
+                                seed_checks_in_area: Dict[int, Dict[str, int]], checks_done: Dict[str, int], slot_data: Dict) -> str:
+
+    from worlds.bingo.Items import item_table
+    ordered_items = multisave.get('received_items', {}).get((team, player, True), [])
+    # web design is my passion
+    text = "<HTML><HEAD><title>Bingo tracker</title><meta http-equiv=\"refresh\" content=\"15\" />"
+    text += "<STYLE>body {font-family: Courier;}</STYLE></HEAD><BODY><table><tr>"
+    for cardnum in range(0, len(slot_data["cards"])):
+        text += "<td>|--------------|<BR>"
+        text += "|B&nbsp;&nbsp;I&nbsp;&nbsp;N&nbsp;&nbsp;G&nbsp;&nbsp;O&nbsp;|<br>"
```

Comment:
probably better use css letter-spacing or pre tag ("preformatted text") depending on the style

Reply:
This is more of a proof of concept. But I am very limited in my knowledge/experience with CSS and web design generally. I would appreciate any help I could get on this